app/pages/SchoolPage.js

```diff
@@ -60,7 +62,7 @@ class SchoolPage extends React.Component {
           <SchoolTitle name={this.state.mainName} />
```

Comment:
The Crome console shows the following error:
Warning: Failed propType: Required prop `official_name` was not specified in `SchoolTitle`. Check the render method of `SchoolPage`.